src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,72 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT = "result";
+    private static final String DELIMITER = ",";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        try (BufferedReader dataFromFile = readFile(fromFileName)) {
+            String report = createReport(calculateSummary(dataFromFile));
+            writeReportToFile(report, toFileName);
+        } catch (IOException e) {
+            System.out.println("Can`t read data from file " + fromFileName + e);
+        }
+    }
+
+    private BufferedReader readFile(String fromFileName) throws FileNotFoundException {
+        return new BufferedReader(new FileReader(fromFileName));
+    }
```

Comment:
this method should read the file and return the read result. But instead you're simply creating beffered reader and pass it to other methods. Let's fix it